docs/zh/maintain/diagnose.md

```diff
@@ -99,7 +100,13 @@ positional arguments:
 
 #### 离线检查
 
-离线检查会检查集群中的离线任务，并输出最终状态为失败的任务（不检查“运行中”的任务）及其日志，等价于`SHOW JOBS`并筛选出失败任务。
+离线检查会输出最终状态为失败的任务（不检查“运行中”的任务），等价于`SHOW JOBS`并筛选出失败任务。
+
+#### JOB 检查
+
+JOB 检查会检查集群中的离线任务，可以使用`inspect job`或`inspect job --state all`查询所有任务，等价于`SHOW JOBS`并按job_id排序。使用`inspect job --state <state>`可以筛选出特定状态的日志，可选的state有`finished running failed stopped killed`等。可以使用 ',' 分隔，同时查询不同状态的日志。
```

Comment:
So the state==job state? If we want to get all error jobs(failed or stopped or lost...), we should exec 'inspect job --state a,b,c'?

Reply:
state can be https://spark.apache.org/docs/3.2.1/api/java/org/apache/spark/launcher/SparkAppHandle.State.html, and the diy state `pending`, `stopped`, you can find in code by `setState`.